tools.yml

```diff
@@ -473,6 +537,7 @@ tools:
     cores: 10
     mem: 90
   toolshed.g2.bx.psu.edu/repos/devteam/velvet/velvetg/.*:
+    cores: 4
```

Comment:
velvetg does not seem to use GALAXY_SLOTS at all

Reply:
I think that if a tool cannot use multiple cores, it would be better to set it to 1 explicitly. The reason is that if someone were to override the defaults, and increase the number of cores, even tools that can't  take advantage of multiple cores would have their core values bumped up.